mav_trajectory_generation_ros/src/ros_visualization.cpp

```diff
@@ -172,7 +172,7 @@ void drawVertices(const Vertex::Vector& vertices, const std::string& frame_id,
   visualization_msgs::Marker& marker = marker_array->markers.front();
 
   marker.type = visualization_msgs::Marker::LINE_STRIP;
-  marker.color = mav_visualization::Color::Chartreuse();
+  marker.color = mav_visualization::Color::Green();
```

Comment:
You don't like Chartreuse :question:

Reply:
Hahaha no. ;) I usually have my background set to white so I can barely see it. Green is a lot easier to see on any background color (other than green).